vtex-sites/base.store
# feat: Add GTM

## What's the purpose of this pull request?
Adds Google Analytics via Google Tag Mager(GTM). The GTM is added via the [`gatsby-plugin-google-tagmanager`](https://www.gatsbyjs.com/plugins/gatsby-plugin-google-tagmanager/).

## How it works? 
<em>Tell us the role of the new feature, or component, in its context.</em>

## How to test it?
Open the dev tools, navigate inside the page and see the `collect` requests on the network tab.
![image](https://user-images.githubusercontent.com/15680320/140947391-00763f2a-3f75-4e05-ba36-288051b476de.png)
 

## References
https://developers.google.com/tag-manager/quickstart

<em>Extra tip: add references to related issues or mention people important to this PR may be good for the documentation and reviewing process</em>
